uwblueprint/planet-read
# Disable filter and display no stories in “Browse stories” if user has…

… no approved languages

## Notion ticket link



[Disable filter and display no stories in “Browse stories” if user has no approved languages](https://www.notion.so/uwblueprintexecs/Disable-filter-and-display-no-stories-in-Browse-stories-if-user-has-no-approved-languages-6befb682fc4f42df85feef89460b16d1)



## Implementation description

- Disable filter and display "No Stories Found" box on browse stories page if user has no approved languages for translation or review



## Steps to test

1. Sign up as someone new or login as someone with no approved languages
2. Click on browse stories tab
3. Observe no filters and "No Stories Found" box
4. Please check that other tabs still work
5. Please check that browse stories tab still works for users with approved languages (e.g. Dwight Translator English (US))



## What should reviewers focus on?

- Does it work
- Does the code look ok

## Checklist
